### PR TITLE
added cards

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
+*{
```
Comment (gauravsinhaweb) [inline cards/cards.css]: use color variables inside :root tag

File: cards/cards.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+
+          <div class="component-details">
+            <h2 class="component-heading">Horizontal Cards</h2>
+            <p class="component-info">A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options.</p>
+        </div>
+        <div class="card-examples flex-flow-row">.
+            <div class="card card-component flex-flow-row card-horizontal">
+                <img class="card-img card-img-horizontal" src="../images/avatar.jpg">
```
Comment (mehulsatardekar) [inline cards/cards.html]: Use alt attribute in image tags

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: cards/cards.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+              <div class="component-details">
+                <h2 class="component-heading">Card with text overlay</h2>
+                <p class="component-info">A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options.</p>
+            </div>
+            <div class="card-examples flex-flow-row">.
+                <div class="card card-component">
+                    <div class="text-overlay">
+                        <h2>Out of stock</h2>
+                    </div>
+                    <img class="card-img" src="../images/avatar.jpg">
```
Comment (mehulsatardekar) [inline cards/cards.html]: Use alt attribute in img tag

**Comment by mehulsatardekar** [comment]
ts
The card would look good if you add some space between texts.

File: cards/cards.html (hunk 3 above)
Comment (swstica) [reply cards/cards.html]: Made all the changes, thanks:)

File: cards/cards.css (hunk 1 above)
Comment (swstica) [reply cards/cards.css]: done

**Pull request merged**